### DISCUSSION TITLE
Unavailable for your site because your domain is not properly configured to support HTTPS

### DISCUSSION THREAD

**Discussion by davorg** [community/community#58454] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Both my apex domain (https://sowman.org/) and a subdomain (https://will.sowman.org/) are hosted on GitHub Pages. The apex domain works fine, but the settings page for the subdomain says that HTTPS is "Unavailable for your site because your domain is not properly configured to support HTTPS". It works when I remove the custom domain, but the problem reappears whenever I add it back.

I can't see anything obvious wrong with the DNS settings, but it's possible I've missed something here.

Repos:

* https://github.com/davorg/sowman.org - apex domain
* https://github.com/davorg/will-sowman - subdomain

**Comment by airtower-luna** [comment, marked as answer]
The documentation using subdomains says that the CNAME record for the subdomain should point at your `github.io` domain, but it's actually pointing at your apex domain. I assume that's the problem, see: https://docs.github.com/en/pages/configuring-a-custom-domain-for-your-github-pages-site/managing-a-custom-domain-for-your-github-pages-site#configuring-a-subdomain

Unrelated, I also noticed your apex domain is lacking AAAA records, so it won't be available over IPv6.

**Reply by davorg** [reply]
> The documentation using subdomains says that the CNAME record for the subdomain should point at your github.io domain, but it's actually pointing at your apex domain. I assume that's the problem, see: https://docs.github.com/en/pages/configuring-a-custom-domain-for-your-github-pages-site/managing-a-custom-domain-for-your-github-pages-site#configuring-a-subdomain

That was because I had been playing around with that value in order to make it work. The actual fix was that the value needed to be `davorg.github.io.` (with a final got) and not `davorg.github.io` (which is what I originally had). Thanks for getting me to look in that direction again.

> Unrelated, I also noticed your apex domain is lacking AAAA records, so it won't be available over IPv6.

Yep. That's a project for the (not too distant) future.
